tests/modules/emails/test_email.py

```diff
@@ -79,3 +81,16 @@ def test_email_templates():
         except:  # NOQA
             print(f'Email Template: {path}')
             raise
+
+
+def test_sender():
+    msg = Email(recipients=[test_recipient])
+    assert msg.sender == 'Codex Mailbot <changeme@example.com>'
+
+    SiteSetting.set_key_value('email_default_sender_name', 'Site Admin')
+    SiteSetting.set_key_value('email_default_sender_email', 'admin@codex.org')
```

Comment:
`codex.org` is a real domain -- maybe use a subdomain of `example.org`, e.g. `codex.example.org`.  just to make sure we dont accidentally bother a real domain.

Reply:
Ah of course, I'll change it

Reply:
btw, [RFC2606](https://datatracker.ietf.org/doc/html/rfc2606#section-3) specifically sets aside these as usable for testing, so no mailer will do anything sketchy while using these.  in theory. haha